force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -102,16 +126,20 @@ public with sharing class ClassesObjectsMethods {
      */
     public static Integer countVowels(String str) {
         // Initialize the result Integer
+        Integer count = 0;
+
+        // get individual characters from the string - hint use substring or split
+        List<String> characters = str.toLowerCase().split('');
 
-        /*
-        for () {
-            // get individual characters from the string - hint use substring or split
+        for (String character : characters) {
             // check if the character is a vowel
-            // if it is a vowel, increment the result Integer
+            if (character == 'a' || character == 'e' || character == 'i' || character == 'o' || character == 'u') {
+                // if it is a vowel, increment the result Integer
+                count++;
+            }
         }
-        */
-
-        return null; // Replace null with the variable you used to store the result
+        
+        return count; // Replace null with the variable you used to store the result
```

Comment:
Good work for problems where I am seeking a membership in a set. In this case, the set of vowels. I like to use a Set rather than conditional logic. 

```
 public static Integer countVowels(String str) {
        Integer vowelCount = 0;
        Set<String> vowels = new Set<String>{
                'a', 'e', 'i', 'o', 'u'
        };
        List<String> characters = str.split('');
        for (String character : characters) {
            if (vowels.contains(character.toLowerCase())) {
                vowelCount++;
            }
        }
        return vowelCount;
    }
 ```